### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by mschulze0** [Nick-MRINZ/dashboard_icons#1]
Hi, unfortunately the module isn't working for me.

```
The 'dashboard_icon_switch' module threw the following exception when calling the hook method 'redcap_every_page_top':

ArgumentCountError: Too few arguments to function MRINZ\dashboard\dashboard::redcap_every_page_top(), 1 passed in redcap/redcap_v11.3.3/ExternalModules/classes/ExternalModules.php on line 2798 and exactly 6 expected in redcap/modules/dashboard_icon_switch_v1.0.1/dashboard.php:15
Stack trace:
#0 redcap/redcap_v11.3.3/ExternalModules/classes/ExternalModules.php(2798): MRINZ\dashboard\dashboard->redcap_every_page_top()
#1 redcap/redcap_v11.3.3/ExternalModules/classes/ExternalModules.php(2926): ExternalModules\ExternalModules::startHook()
#2 redcap/redcap_v11.3.3/ExternalModules/classes/ExternalModules.php(2959): ExternalModules\ExternalModules::ExternalModules\{closure}()
#3 redcap/redcap_v11.3.3/Classes/Hooks.php(42): ExternalModules\ExternalModules::callHook()
#4 redcap/redcap_v11.3.3/Classes/HtmlPage.php(127): Hooks::call()
#5 redcap/redcap_v11.3.3/Views/HeaderProject.php(25): HtmlPage->PrintHeader()
```

**Comment by mschulze0** [comment]
`void redcap_every_page_top ( int $project_id )` only takes one argument..not sure why it would work for you with 6 arguments

**Comment by mschulze0** [comment]
Upon removing the last 5 arguments it's working now :)

But for some reason in the legend for status icons it doesn't show the entry for "complete"

**Comment by Nick-MRINZ** [comment]
Thanks for spotting that don't know how I missed that or why it was working on my instance with those arguments. 
I'm unsure why the complete icon in the legend isnt showing for you.
What version are you on?

**Comment by mschulze0** [comment]
> Thanks for spotting that don't know how I missed that or why it was working on my instance with those arguments.
> I'm unsure why the complete icon in the legend isnt showing for you.
> What version are you on?

11.3.3

this is what the legend looks for me:

![image](https://user-images.githubusercontent.com/43603448/134673724-19f97bc0-96ee-46a3-a89c-56cb8831f841.png)

**Comment by Nick-MRINZ** [comment]
Have fixed the legend issue in the latest release 1.0.3. I was dropping the last row of the table to add custom rows but didn't take into account that the legend changes depending on it surveys or repeating instruments are enabled.
